Frameworks/UIKit.Xaml/ScrollView.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ScrollView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <!-- Layer content which is a image is behind any sublayers-->
+    <Image x:Name="LayerContentImage"></Image>
```

Comment:
we may want to lazy-create this like we do elsewhere (button, etc.), since it's probably not used very frequently